algorithms/scaling/scaler.py

```diff
@@ -790,14 +793,23 @@ def round_of_outlier_rejection(self):
         """Perform a round of outlier rejection, set a new outliers array."""
         assert self.global_Ih_table is not None
         if self.params.scaling_options.outlier_rejection:
+            E2_outlier_indices = determine_Esq_outlier_index_arrays(
```

Comment:
Could we be consistent in usage of `E2_*` vs `Esq`? Possibly a slight preference for the former, but no strong opinion.

Reply:
Thanks, I've previously used `Esq` in the scaling code so I've now changed to that term that here for consistency